app/models/local_restriction.rb

```diff
@@ -1,4 +1,6 @@
 class LocalRestriction
+  OUT_OF_DATE = false
```

Comment:
This doesn't really feel the domain of the restriction, given it's not the restriction that is out of date it's the whole system - for something quick it'd probably make most sense as a helper in the controller.

Reply:
I um'ed and ah'ed about where to put this. A helper in the controller was one of the options I tried, but wasn't sure if that was appropriate. I picked LocalRestriction because it is the restriction data that is out of date, but I'm happy to move it.

Reply:
I'm not sure how to stub the helper method in the integration tests. That is another bonus of putting the flag in the model!